contracts/governance/Governor.sol

```diff
@@ -424,37 +431,44 @@ abstract contract Governor is Context, ERC165, EIP712, IGovernor, IERC721Receive
         }
     }
 
+    /**
+     * @dev Internal queuing mechanism.
+     *
+     * NOTE: This reverts by default, and must be overriden (without calling super) to implement queuing.
+     */
+    function _queue(
+        uint256 /* proposalId */,
+        address[] memory /* targets */,
+        uint256[] memory /* values */,
+        bytes[] memory /* calldatas */,
+        bytes32 /*descriptionHash*/
+    ) internal virtual {
+        revert GovernorQueueNotImplemented();
```

Comment:
So the GovernorTimelock modules should override both the `_queue` function and the `_queuingEnabled` function ?

Reply:
>(both will have to be "resolved" in the final contracts :/)

I think this is only if you explicitly inherit both Governor and GovernorTimelockControl. We have to experiment with that a bit...

Reply:
In a perfect world we would be able to detect if queuing was enabled by comparing function pointers:

```solidity
bool isQueuingEnabled = _queue == Governor._queue;
```

Reply:
I'm wondering, since `_queueEnabled` will not call super when overriden, do we not have a similar issue here? 

What about `_cancel` returning a bool ?

Reply:
I implemented the boolean return